database/factories/UserFactory.php

```diff
@@ -26,8 +26,12 @@ public function definition()
 
         return [
             'name' => $name,
-            'slug' => Str::slug($name),
-            'email' => $this->faker->unique()->safeEmail(),
+            'slug' => function (array $attributes) {
+                return Str::slug($attributes['name']);
+            },
+            'email' => function (array $attributes) {
+                return Str::slug($attributes['name']).'@'.$this->faker->safeEmailDomain();
+            },
```

Comment:
I assume that `$attributes` is basically to the configuration array. Does this mean that `$name` isn't necessary anymore and the value can be directly assigned in the array?